api/src/wfl/service/postgres.clj

```diff
@@ -81,22 +83,24 @@
     (make-update-workflows get-terra-status)))
 
 (defn batch-update-workflow-statuses!
-  "Use `tx` to update `status` the workflows in a `workload`."
-  [tx {:keys [executor uuid items]}]
-  (let [uuid->status (->> {:label (str "workload:" uuid) :includeSubworkflows "false"}
-                          (cromwell/query (first (all/cromwell-environments executor)))
+  "Use `tx` to update the `status` of the workflows in `_workload`."
+  [tx {:keys [executor items uuid] :as _workoad}]
+  (let [env (-> executor all/cromwell-environments first)
+        uuid->status (->> {:label               (str "workload:" uuid)
+                           :includeSubworkflows "false"}
+                          (cromwell/query env)
```

Comment:
Nice catch! I fixed this by passing in the URL as I changed the whole `cromwell` module.

Reply:
Will probably want something similar for the Clio service.